.github/actions/pub-release/action.yaml

```diff
@@ -35,12 +35,27 @@ runs:
       shell: bash
       run: echo "$GITHUB_WORKSPACE/mobile-tools/bin" >> $GITHUB_PATH
 
+    - name: Extract package name from pubspec
+      id: extract_package_name
+      uses: mikefarah/yq@master
+      with:
+        cmd: cat ${{ inputs.path }}/pubspec.yaml | yq '.name'
```

Comment:
I thought it would break if `inputs.path` would be, for example, `packages/patrol_cli/` instead of `packages/patrol/cli`, but [fortunately it won't](https://stackoverflow.com/a/24026057/7009800).